TCMSApp/src/client/app/blocks/third-party/moment.factory.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
let's skip these kind of comments. everybody know whose code it is because of git.
you can override the basic view of new js file in settings->templates->js